services/rewards/service.go

```diff
@@ -168,3 +180,15 @@ func (s *Service) GetParameters(ctx context.Context, currency *BaseCurrency) (*P
 
 	return params, nil
 }
+
+type CardBytes []byte
+
+func (s *Service) GetCardsAsBytes(ctx context.Context) (CardBytes, error) {
+	out, err := s.s3Svc.GetObject(ctx, &s3.GetObjectInput{Bucket: &s.cfg.Cards.Bucket, Key: &s.cfg.Cards.Key})
```

Comment:
A multitude of reasons -
1. The [s3Client interface](https://github.com/brave-intl/bat-go/blob/8a3b0da75900425376dc932f24d091e60c48d95d/libs/aws/s3.go#L17) should be declared where it is used and not in the libs package (Go convention)
2. We want to remove any dependencies on the shared lib where we can

Reply:
👍🏻 makes sense - do we have an issue open for that? Just want to make sure we get some time to clean it up later rather than getting the codebase split on 2 different conventions (also this isn't blocking, just offering suggestions based on what I've seen in the codebase)